cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -359,9 +360,18 @@ function algoliaLocalizedProduct(parameters) {
 
         for (var i = 0; i < attributeList.length; i += 1) {
             var attributeName = attributeList[i];
+            var attributeNameObj = attributeName.split('.');
```

Comment:
Naming: since this is an array and not an object, it's better to have the name reflect it so it's less confusing below when reading the code
```suggestion
            var attributeNameArr = attributeName.split('.');
```